kyuubi-server/src/main/scala/org/apache/kyuubi/server/TempFileService.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.kyuubi.service
+package org.apache.kyuubi.server
```

Comment:
How about keeping the `TempFileService` class in the `org.apache.kyuubi.service` package?